packages/cli/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
+
+## Recent Changes
+
+- BugFix: The `zowe zos-files copy data-set` command overwrites the contents of the target data set without user confirmation. A `--safe-replace` option was added which prompts the user to confirm before overwriting the contents of the target data set. [#2369] (https://github.com/zowe/zowe-cli/issues/2369)
```

Comment:
since `--safe-replace` is a 2 word option... would it be ok to add an alias `--sr` ? 😋 
If so, we will need to update some snapshots 😢